plugins/quota/actor.go

```diff
@@ -0,0 +1,68 @@
+package quota
+
+import (
+	"os"
+	"path"
+
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+	gderrors "github.com/gluster/glusterd2/pkg/errors"
+)
+
+type quotadActor struct{}
+
+const (
+	quotaDaemonKey = "enable"
+)
+
+func (actor *quotadActor) Do(v *volume.Volinfo, key string, value string) error {
+	if key != quotaDaemonKey {
+		return nil
+	}
+	quotadDaemon, err := NewQuotad()
+	if err != nil {
+		return err
+	}
+
+	switch value {
+	case "on":
+
```

Comment:
Just a suggestion:

For this repeated pattern where error is the only value returned, there's a short notation in golang:

```go
if err := os.MkdirAll(path.Dir(quotadDaemon.pidfilepath), os.ModeDir|os.ModePerm); err != nil {
    return err
}
```